docs/index.yml

```diff
@@ -1,111 +1,132 @@
-### YamlMime:Landing
+### YamlMime:Hub
 
 title: Office Add-ins documentation # < 60 chars
 summary: Use the Office Add-ins platform to build solutions that extend Office applications and interact with content in Office documents and in Outlook mail messages and calendar items. With Office Add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build solutions that can run in Office on the web, Windows, Mac, Android, and iOS. # < 160 chars
```

Comment:
Do we want to explicitly call out Android here?

Reply:
Maybe we can replace Android and iOS with mobile?

Reply:
Yeah, that's works. I'd also be okay with "web, Windows, Mac, and more", since the "and more" varies per application.

Reply:
This thing really should aim to be a LOT smaller of a sentence. Let's revisit at a team meeting because I think the reco here is 160 chars. This is leftover from the previous yaml from the landing page